ChairManager/ChairManager.cpp

```diff
@@ -1659,37 +1663,18 @@ void ChairManager::EnableMod(std::string modName, bool enabled) {
     }
 }
 
-bool ChairManager::DeployMods() {
-    mergeXMLFiles();
-    saveChairloaderConfigFile();
-    if(packChairloaderPatch()){
-        if(copyChairloaderPatch()){
-            m_DeployLogMutex.lock();
-            m_DeployState = DeployState::Done;
-            m_DeployLogMutex.unlock();
-            overlayLog(severityLevel::info, "Mods Deployed");
-//            m_State = State::MainWindow;
-            return true;
-        }
-    }
-//    m_State = State::MainWindow;
-    return false;
-}
-
 void ChairManager::mergeXMLFiles(bool onlyChairPatch) {
     // Remove old output files
     m_DeployLogMutex.lock();
-    m_DeployState = DeployState::RemovingOldOutput;
-    m_DeployLogMutex.unlock();
+        m_DeployLogMutex.unlock();
```

Comment:
Mutex is locked and immediately unlocked? Same in multiple places nearby

Reply:
fixed, additionally removed alot of the old deprecated code